logback-ecs-encoder/src/main/java/co/elastic/logging/logback/EcsEncoder.java

```diff
@@ -47,7 +47,7 @@
     private boolean includeMarkers = false;
     private ThrowableProxyConverter throwableProxyConverter;
     private boolean includeOrigin;
-    private List<Pair> additionalFields = new ArrayList<Pair>();
+    private final List<Pair> additionalFields = new ArrayList<>();
```

Comment:
This fails the build as diamond operators are not allowed when the bytecode level is 6.
```suggestion
    private final List<Pair> additionalFields = new ArrayList<Pair>();
```